composer.json

```diff
@@ -19,7 +19,8 @@
         "symfony/console": "^2.7|^3.0|^4.0",
         "symfony/http-foundation": "^2.7|^3.0|^4.0",
         "symfony/security": "^2.7|^3.0|^4.0",
-        "bugsnag/bugsnag": "^3.15.0"
+        "bugsnag/bugsnag": "^3.15.0",
+        "symfony/messenger": "^4.3"
```

Comment:
This component cannot be installed with Symfony < 4.3:

composer conflicts:
- symfony/event-dispatcher: <4.3
- symfony/debug: <4.1

Reply:
Shouldn't it be a dev dependency?